src/clj/game/core/abilities.clj

```diff
@@ -15,12 +15,12 @@
     (should-trigger? state side (make-eid state) card targets ability))
   ([state side eid card targets {:keys [req optional psi trace] :as ability}]
    (when ability
-     (let [should-trigger? (partial should-trigger? state side eid card targets)]
+     (let [s-t? (partial should-trigger? state side eid card targets)]
        (cond
          req (req state side eid card targets)
-         optional (should-trigger? optional)
-         psi (should-trigger? psi)
-         trace (should-trigger? trace)
+         optional (s-t? optional)
+         psi (s-t? psi)
+         trace (s-t? trace)
```

Comment:
I'd rather the partial `should-trigger` be called `partial-should-trigger?` than `s-t?` since the latter give no information what it means (even though it should be clear from the let).